lab_4_retrieval_w_clustering/main.py

```diff
@@ -51,6 +59,14 @@ def set_tokenized_corpus(self, tokenized_corpus: TokenizedCorpus) -> None:
         Raises:
             ValueError: In case of inappropriate type input argument or if input argument is empty.
         """
+        if not tokenized_corpus or not isinstance(tokenized_corpus,list):
+            raise ValueError
+        self._corpus = tokenized_corpus
+        total = 0
+        for i in self._corpus:
+            total += len(i)
+        self._avg_doc_len = float(total) / float(len(self._corpus))
```

Comment:
is it really need to convert to float?

Reply:
Not really, was just doing it to be sure

Reply:
done a long time ago!